oclif.manifest.json

```diff
@@ -1 +1,2950 @@
\ No newline at end of file
+{
```

Comment:
it looks like your oclif manifest is making the `args` objects instead of arrays too (so args are not working properly)
ex:
<img width="778" alt="image" src="https://github.com/DevCycleHQ/cli/assets/98763537/9c0263bd-7dd0-414a-8f01-7591ac95d7b3">

Reply:
I did not change the cleanup command, and I believe kaushal's ticket should take care of the args not working as expected?